RealmSwift/Tests/RealmCollectionTypeTests.swift

```diff
@@ -904,90 +904,6 @@ class ListStandaloneRealmCollectionTypeTests: ListRealmCollectionTypeTests {
         assertThrows(collection.filter("ANY stringListCol == %@", CTTStringObjectWithLink()))
     }
 
-    override func testMin() {
```

Comment:
No, they're no longer being overridden from the base class because aggregates are now supported for unmanaged objects.